### DISCUSSION TITLE
My device network is normal, but the command 'briefcase new' encountered the following error.

### DISCUSSION THREAD

**Discussion by jlij** [beeware/toga#2820] [Q&A] [answered]
[flow] Generating a new application 'flow'
Using app template: https://github.com/beeware/briefcase-template, branch v0.3.19
Cloning template 'https://github.com/beeware/briefcase-template'...

Unable to clone repository 'https://github.com/beeware/briefcase-template'.

This may be because your computer is offline, or because the repository URL is incorrect.
[briefcase.2024_09_07-08_06_33.new.log](https://github.com/user-attachments/files/16913771/briefcase.2024_09_07-08_06_33.new.log)

**Comment by rmartin16** [comment, marked as answer]
Please try upgrading your installed version of Git.

```
GitCommandError: Cmd('git') failed due to: exit code(129)
  cmdline: git clone -v --filter=blob:none --no-checkout -- https://github.com/beeware/briefcase-template /Users/[...]/Library/Caches/org.beeware.briefcase/templates/briefcase-template
  stderr: 'error: unknown option `filter=blob:none'
```

The next version of Briefcase will provide a [better failure mode](https://github.com/beeware/briefcase/pull/1915) for this :)
